internal/transform/asset.go

```diff
@@ -44,11 +45,14 @@ func transformSingleAsset(asset xdr.Asset) (AssetOutput, error) {
 		return AssetOutput{}, fmt.Errorf("Unable to hash asset for payment operation")
 	}
 
+	farmAssetID := FarmHashAsset(outputAssetCode, outputAssetIssuer, outputAssetType)
+
 	return AssetOutput{
 		AssetCode:   outputAssetCode,
 		AssetIssuer: outputAssetIssuer,
 		AssetType:   outputAssetType,
 		AssetID:     outputAssetID,
+		ID:          farmAssetID,
```

Comment:
Having two asset ids on the `history_assets` table might be confusing. We might want to consider dropping the original `AssetID` from the table in the future.

Reply:
I totally agree. Two IDs doesn't make any sense. We should drop it as soon as possible.